lib/LTP/utils.pm

```diff
@@ -432,4 +421,21 @@ sub install_from_repo {
     }
 }
 
+sub prepare_whitelist_environment {
```

Comment:
Please add this to `@EXPORT` so you don't need to prefix it with `LTP::utils::`.